protocols/twitter/twitter_lib.c

```diff
@@ -602,6 +604,90 @@ static gboolean twitter_xt_get_status_list(struct im_connection *ic, const json_
 	return TRUE;
 }
 
+static char *twitter_msg_get_text(struct im_connection *ic, int log_id, int reply_to,
+				struct twitter_xml_status *txs, const char *prefix) {
+	gchar * format = set_getstr(&ic->acc->set, "twitter_format_string");
+	gchar * rt_format = set_getstr(&ic->acc->set, "twitter_retweet_format_string");
+	gchar * reply_format = set_getstr(&ic->acc->set, "twitter_reply_format_string");
+
+	GString * rt = g_string_new(NULL);
+	GString * reply = g_string_new(NULL);
+	GString * text = g_string_new(NULL);
```

Comment:
Not sure about these three. It's instantiating three GStrings for three different kinds of messages that don't apply to all tweets.